specs-go/config.go

```diff
@@ -31,6 +31,8 @@ type Spec struct {
 type Process struct {
 	// Terminal creates an interactive terminal for the container.
 	Terminal bool `json:"terminal,omitempty"`
+	// InitialConsoleSize contains the initial h,w of the console.
+	InitialConsoleSize [2]uint `json:"initialConsoleSize"`
```

Comment:
+1 for explicit height/width.  Although probably no need for the specific name.  `type Box` would give us something we could use in other places, although I can't think of other consumers now.